pulp_2_tests/tests/rpm/api_v2/test_copy.py

```diff
@@ -73,11 +77,14 @@ def test_all(self):
         self.addCleanup(client.delete, repos[1]['_href'])
 
         # Copy data to second repository.
-        client.post(urljoin(repos[1]['_href'], 'actions/associate/'), {
-            'source_repo_id': repos[0]['id'],
-            'override_config': {'recursive': True},
-            'criteria': {'filters': {}, 'type_ids': ['erratum']},
-        })
+        client.post(
+            urljoin(repos[1]['_href'], 'actions/associate/'),
+            {
+                'source_repo_id': repos[0]['id'],
+                'override_config': {'recursive': True},
+                'criteria': {'filters': {}, 'type_ids': ['erratum']},
+            },
+        )
```

Comment:
These are all just formatting changes, right?

Reply:
Yes.

Reply:
I ran black against this module.

Reply:
The class `MultipleSourceRepositoriesRecursiveCopyTestCase` is the one testing the newly added feature.